Microsoft-foundry-best-practices-intermediate.html

```
@@ -0,0 +1,3560 @@
+          font-size: 13px;
+        }
+      }
+      @media (min-width: 881px) {
+        .menu-btn {
+          display: none;
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <button class="menu-btn" id="menuBtn" aria-label="目次を開く">☰ 目次</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)